.github/workflows/cd.yml

```
@@ -0,0 +1,50 @@
+          pip install -r requirements.txt
+
+      # Step 4: Run tests
+      - name: Run Tests
+        run: make test
+
+      # Step 5: Set up Google Cloud SDK
+      - name: Set up Google Cloud SDK
+        uses: google-github-actions/setup-gcloud@v1
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
```

actionlint (1.7.3)

input "service_account_key" is not defined in action "google-github-actions/setup-gcloud@v1". available inputs are "install_components", "project_id", "skip_install", "version"

(action)